src/System.Management.Automation/namespaces/Win32Native.cs

```diff
@@ -175,720 +175,5 @@ internal enum SECURITY_IMPERSONATION_LEVEL : short
         // Security Quality of Service flags
         internal const int SECURITY_ANONYMOUS = ((int)SECURITY_IMPERSONATION_LEVEL.Anonymous << 16);
         internal const int SECURITY_SQOS_PRESENT = 0x00100000;
-
-#if !CORECLR // Only enable/port what is needed by CORE CLR.
```

Comment:
Not so sure that they should be removed. There are transaction related APIs here. We will need them if we want to enable transaction support in powershell core.